setup.cfg

```
@@ -56,7 +56,7 @@ python_requires = >=3.7
 include_package_data = True
 install_requires =
     metomi-rose>=2.1.0, <2.3.0
-    cylc-flow==8.2.*
+    cylc-flow>=8.2.6, <8.3
```

Try this so that 8.2.6.dev0 gets allowed
```suggestion
    cylc-flow>8.2.5, <8.3
```